### PR TITLE
Address a variety of usability fixes, closes #17

### LINKED ISSUE #17: Usability suggestions
Hey, I decided to take note of a handful of usability improvements that could be made to the site:

- Font size is 10px on mobile in some areas, which doesn't meet minimum accessibility standards. We want our base font size to be about 15px
- Markers currently do not display the name of the place but this is a pretty straightforward adjustment.
- Popups do not scroll, which can be problematic for some of the popups with a lot of content on small screens.
- close and hamburger menu hit areas are too small on mobile (should be 40px minimum width/height).

I will have time to address these issues tonight unless someone gets there before me – happy to hear other suggestions as well :)

### PR DESCRIPTION
**Pull request by samanpwbb**
Screen Shot 2020-06-02 at 1 05 04 AM

This PR does the following things:
- Adds normalize.css to make sure things look similar across browsers
- Reworks the layout and visual styles in the app
- Increases font sizes across the board
- Enables popups to scroll if above a certain size.
- Redesigns the sidebar interaction to no longer use a hamburger button
- Removes the visitor counter (unsure of what the purpose of this was – I can bring it back if it's important)
- Fixes the flyTo animation when selecting a location from the location list
- Ensures all buttons have appropriate sized hit areas

This PR _does not_ do the following things:
- Add labels to the basemap
- Filter markers when filters are selected

Lets address these issues in future PRs since they both require more significant refactors. 

This gif shows off most of the changes on mobile, the primary place where the app will be used:

@jdalt for review plz 🙏

### CODE REVIEW

**Comment by jdalt** [comment]
Hot take: love it. Unified sidebar solves a lot of the responsive ugliness going on. The counter was desired by the lead organizer and it's like a baby step toward analytics and answering how people are using this. I have more thoughts bubbling on this but not enough time to voice them atm. Hope to fully address by afternoon.

File: script.js [hunk 1]
```diff
@@ -44,11 +44,13 @@ let locations = []
 // See also: https://docs.mapbox.com/mapbox-gl-js/example/setstyle/
 const map = new mapboxgl.Map({
   container: 'map',
-  style: 'mapbox://styles/saman/ckawvg6bk011x1ipepu7nqlbh',
+  style: 'mapbox://styles/saman/ckawvg6bk011x1ipepu7nqlbh?fresh=true',
```
Comment (jdalt) [inline script.js]: Knocks out some caching?

File: normalize.css [hunk 2]
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (jdalt) [inline normalize.css:1]: Thank you for bringing this. Not something I bring to the project.

File: styles.css [hunk 3]
```diff
@@ -1,113 +1,252 @@
-body { margin: 0; padding: 0; }
-#map { position: absolute; top: 0; bottom: 0; width: 100%; }
```
Comment (jdalt) [inline styles.css:2]: So much more more organization here. Very welcome!

**Review by jdalt — COMMENTED** [review]
No complaints. This is ready to go and I really like how this improves the code quality and solves many of the pressing usability concerns.

I'd like to quick test this locally and get some closure on the issue of the little site counter. Might go back to the slack channel on that. Hope to have answers on this by 11:30am.

File: script.js (hunk 1 above)
Comment (samanpwbb) [reply script.js]: Oh I need to remove this param - was getting intermittent 404s due to caching but this param will rate limit us

**Comment by samanpwbb** [comment]
> it's like a baby step toward analytics and answering how people are using this.

I will re-institute the counter but a) you can use your Mapbox account to check usage by map load and referrer – would that be a reasonable alternative to the counter until we have a proper analytics tool set up?

**Comment by samanpwbb** [comment]
Just resolved conflicts with some other changes, here's the latest. Added the visitor counter back to top right and restyled the new urgent notices:

Screen Shot 2020-06-02 at 10 01 13 AM

@jdalt this should be good to go. Tested on ff/safari/chrome, _not_ IE.

**Comment by jdalt** [comment]
merged in new repo https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/pull/1

**Pull request closed without merging**